packages/loot-core/src/server/budget/goaltemplates.js

```diff
@@ -299,12 +297,8 @@ async function applyCategoryTemplate(category, template_lines, month, force) {
         let num_months = differenceInCalendarMonths(to_month, current_month);
         let target = amountToInteger(template.amount);
         if (num_months < 0) {
-          console.log(
-            `${category.name}: ${`${template.to} is in the past:`} ${
-              template.line
-            }`,
-          );
-          return null;
+          errors.push(`${`${template.month} is in the past.`}`);
```

Comment:
```suggestion
          errors.push(`${template.month} is in the past.`);
```

Reply:
Weird.  I didn't realize I did that.

Done.